### PR TITLE
[POC] Viewing individual files

### PR DESCRIPTION
**Pull request by willdurand**
This is a Proof of Concept (POC) to show individual files with syntax highlighting:

### CODE REVIEW

File: src/pages/Browse/index.tsx [hunk 1]
```diff
@@ -68,6 +96,7 @@ const mapStateToProps = (
   return {
     apiState: state.api,
     version: getVersionInfo(state.versions, parseInt(versionId, 10)),
+    file: getVersionFile('', state.versions, parseInt(versionId, 10)),
```
Comment (willdurand) [inline src/pages/Browse/index.tsx]: I believe this is a code smell. We need to store the `path` we click on somehow, no?

Comment (willdurand) [reply src/pages/Browse/index.tsx]: @bobsilverberg ^ (sorry I thought I mentioned you in the previous comment)

Comment (bobsilverberg) [reply src/pages/Browse/index.tsx]: I think this has now been dealt with, via PR #232, correct?

Comment (willdurand) [reply src/pages/Browse/index.tsx]: yep :)

**Comment by willdurand** [comment]
I am closing this PR now as I more or less landed most of the things needed to write the show/select content feature.

**Pull request closed without merging**